### DISCUSSION TITLE
[M-Z] (22 Gennaio 2019) Esercizio 4

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#435] [Esami & Esoneri M-Z]
## Traccia

<img width="932" alt="image" src="https://github.com/user-attachments/assets/922d0c5c-5203-430a-ab9e-b5bedb134ee9">

>**Stesso esercizio:** #426 (15 Gennaio 2018 - Esercizio 4)

**Comment by rimaout** [comment]
<img width="502" alt="image" src="https://github.com/user-attachments/assets/f58fe990-149a-4c10-be8e-f1b1dbc16e4d">

**Comment by matypist** [comment]
Questo esercizio è un duplicato di [[M-Z] (15 Gennaio 2018) Esercizio 4 #426](https://github.com/sapienzastudentsnetwork/metodi-matematici-per-l-informatica/discussions/426) , per evitare una frammentazione delle proposte di soluzione potete quindi scrivere/leggere eventuali possibili soluzioni direttamente lì
